### PR TITLE
WIP: fix pep8 violations

### LINKED ISSUE #4: fix PEP8 violations

### PR DESCRIPTION
**Pull request by kevindaum**
This will close #4.

### CODE REVIEW

File: callisto/delivery/forms.py [hunk 1]
```diff
@@ -97,72 +97,109 @@ def clean_key(self):
...
-    email_confirmation = forms.ChoiceField(choices=[(True, "Yes"), (False, "No, thanks")],
-                                           label="Would you like us to send you a confirmation email with information about your rights in the reporting process, and where to get support and find resources on campus?",
-                                           required=True,
-                                           widget=forms.RadioSelect)
+                             required=True,
+                             max_length=256,
+                             widget=forms.TextInput(attrs={'placeholder': 'ex. myname@gmail.com'}),)
+
+    contact_notes = forms.CharField(
+        label=("Any notes on what time of day is best to reach you? A {school} staff member will"
```
Comment (kevindaum) [inline callisto/delivery/forms.py]: @kelseyq you okay with this way of breaking up long strings? I'm using python's implied line continuation within parentheses, brackets and braces. PEP8 [says this is the preferred method](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).

Comment (kelseyq) [inline callisto/delivery/forms.py]: yup, sounds good!

**Comment by kelseyq** [comment]
This looks fine to me so far. My main concern with code style is that any rules we're enforcing can be checked (& ideally even fixed!) programmatically. Happy to defer to your judgement otherwise.

**Comment by kevindaum** [comment]
Sorry I'm taking so long on this. I thought I'd have more time this summer. Also, since you mentioned you wanted style rules to be fixed programmatically, I started doing a little looking into other options. The only one I've found besides [autopep8](https://pypi.python.org/pypi/autopep8) so far is what's builtin to PyCharm. I'm guessing you'd rather use `autopep8`, which seems to be the engine behind plugins for many editors, over PyCharm, which may be proprietary.

**Comment by kelseyq** [comment]
Thanks for looking into the automated checking, Kevin! I need to merge #68 in for a production deploy today; if you don't mind, I'll take care of the merge from master back into this branch so I can try out autopep8 and fiddle with Travis.

**Comment by kevindaum** [comment]
job__150_7_-_sexualhealthinnovations_callisto-core_-_travis_ci

Hmm. That's not particularly helpful.

**Comment by kelseyq** [comment]
hah, no kidding!

**Comment by kevindaum** [comment]
Oh, never mind, I'm just being hasty. The output from `flake8 .` was above that screen shot. 

And re: you doing the merge and fiddling: 👍🏻

**Comment by kevindaum** [comment]
FYI, the `autopep8` command I used is in the 974d732 commit message. I turned it up to eleven but you can dial it back a little by leaving the one or both of the `--aggressive` flags off.

**Pull request merged**